gramps_webapi/api/resources/emit.py

```diff
@@ -36,7 +56,7 @@ def response(
         payload = payload or {}
         args = args or {}
         if "strip" in args:
-            self.strip_empty_keys = True
+            self.strip_empty_keys = args["strip"]
         else:
             self.strip_empty_keys = False
```

Comment:
Why is this `if`/`else` still needed rather than just `self.strip_empty_keys = args["strip"]`? Perhaps needs adding a default value in the webargs schema.

Reply:
So it continues to work for any endpoint as opposed to only those derived from GrampsObjectResource and GrampsObjectsResource.